### PR TITLE
Scenario Dropdown Updates

### PR DESCRIPTION
**Pull request by arottersman**
## Overview
- Consolidate all the scenario actions into a sub-dropdown menu to the right of the Scenario dropdown menu (removes hiding extra icons behind the ellipses)
- This initially broke the rename action because the button was no longer part of the same view as the editable text. While we could trigger a backbone event for the parent view, and then do the rename there in the same way, it seemed cleaner/faster to just implement #1887 and show a rename popup like the rest of the actions

(Ignore the typo in the branch name 😨 )

Connects #1909 
Connects #1887 

### Demo

c22evlaoj2

## Testing Instructions
 * Pull and `./scripts/bundle.sh`
 * For both models, open a project and confirm all the various scenario actions work
 * Confirm renaming a project works as before

### CODE REVIEW

**Comment by rajadain** [comment]
Some tests are failing

**Comment by arottersman** [comment]
@azavea-bot rebuild

**Comment by arottersman** [comment]
Okay, the build is finally passing!

**Comment by rajadain** [comment]
Taking a look now.

File: src/mmw/js/src/modeling/templates/scenarioMenuItemOptions.html [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    <li class="scenario-options-dropdown-menu-item" role="presentation" data-action="delete">
+        <i class="fa fa-trash" title="Delete"></i> Delete
+    </li>
+{% endif %}
+{% if is_gwlfe %}
+    <li class="scenario-options-dropdown-menu-item" role="presentation" data-action="export-gms">
+        <form id="export-gms-form" method="post" action="/api/modeling/export/gms/" target="_blank">
+            <input type="hidden" name="csrfmiddlewaretoken" value="{{ csrftoken }}">
+            <input type="hidden" name="filename" class="gms-filename" value="export.gms">
+            <input type="hidden" name="mapshed_data" value='{{ gis_data }}'>
```
Comment (rajadain) [inline src/mmw/js/src/modeling/templates/scenarioMenuItemOptions.html]: I know this carried over from the source from which this was copied, but let's prefer double quotes for all HTML attributes. Specifically, `value` uses single quotes which should be changed to double.

**Comment by rajadain** [comment]
This could use feedback from @jfrankl, but in the implementation the second dropdown looks "disconnected" from the first:

image

As opposed to the mockup where they are more "connected":

image

I'm not sure which is better, but wanted to make sure we be intentional about our choice.

**Comment by arottersman** [comment]
I also forgot the grey square around the ellipsis on click which may help the dropdown feel connected to the row

File: src/mmw/js/src/modeling/templates/scenarioMenuItemOptions.html (hunk 1 above)
Comment (arottersman) [inline src/mmw/js/src/modeling/templates/scenarioMenuItemOptions.html]: Double quotes broke the template. I think`gis_data` has characters that need to be escaped in it. I'll add a comment

**Comment by arottersman** [comment]
Gouda?
w7gpemasst

**Comment by rajadain** [comment]
Cheese 👍

File: src/mmw/js/src/modeling/templates/scenarioMenuItemOptions.html (hunk 1 above)
Comment (rajadain) [inline src/mmw/js/src/modeling/templates/scenarioMenuItemOptions.html]: Ah 🤦‍♂️ serves me right for being too nitpicky. And of course I'm the one who wrote this originally https://github.com/WikiWatershed/model-my-watershed/commit/2834f5ba0093ca968d1fe9fe744d2e0a9687c567

**Comment by arottersman** [comment]
There's a bug on FF (IE11 + Safari are fine though)
screen shot 2017-06-23 at 9 55 52 am

**Comment by arottersman** [comment]
I pushed a small fixup to address. @rajadain mind giving this a quick final glance before I merge?

**Comment by rajadain** [comment]
On it

**Pull request merged**